### ISSUE TITLE
Select and Cancel buttons are not Working. 

### ISSUE THREAD

**Issue by shafa909** [sdoomz/react-google-picker#38]
https://user-images.githubusercontent.com/23289583/167604258-9d53d2bb-b561-470f-98c4-ae5a0278a196.mov

**Comment by shafa909** [comment]
I can select the file and getting data by double clicking the file.

stackoverflow link : https://stackoverflow.com/questions/72195696/submit-and-cancel-button-not-working-in-react-google-picker

**Comment by paustint** [comment]
Anyone ever find a solution? I am not using this library but I have the exact same issue in my react application....

I can select a file by double clicking, but that is the only way to select.
I have a "select folder" option in my app and there is no way to select a folder since double click does not work there.

**Comment by paustint** [comment]
I have been plagued with this issue for a long time (somewhat intermittently) but now it is very consistent.

I finally figured out the cause of my issue, may or may not be the same as others - but I had a modal open to allow the user to choose where to save a file and if they opened the google picker to choose a folder it would not work - but it seemed to work fine in other places.

Not sure exactly why, but my modal seemed to have a conflict with the google drive picker modal. Select/cancel/X/filter menus all did not work. I had to keep track of when the google picker was open and omit some of the CSS classes for my modal and it appeared to resolve the issue.

In summary - I think there was some css conflict from items on my page that only applied to a few components on the picker.

I was not using this exact library so I was able to access the picker instance and check if it was visible - YMMV https://github.com/jetstreamapp/jetstream/pull/176
